app/services/include_params_validator.rb

```diff
@@ -6,7 +6,7 @@ class IncludeParamsValidator
   attr_reader :relationships, :supported_relationships
 
   validates_each :relationships, allow_blank: true do |record, _attr, values|
-    unsupported_values = values - record.supported_relationships
+    unsupported_values = values - record.formatted_supported_relationships
```

Comment:
I am not sure about the name,
*formatted* conveys a different meaning, like formatting a text,
this case is more related to: extended , expanded, spread, splitted, ... you got the point :)

Reply:
['split', 'splat', 'explicitify'].product(['supported_relationships', 'nested_chains']).map { |t| t.join('_') }
=> ["split_supported_relationships",
 "split_nested_chains",
 "splat_supported_relationships",
 "splat_nested_chains",
 "explicitify_supported_relationships",
 "explicitify_nested_chains"]

Reply:
Maybe pick one of ^^

Reply:
Also, I think format is basically fine

```
verb
(especially in computing) arrange or put into a format.
"how to format a document"
```